.github/workflows/pythonpublish.yml

```diff
@@ -24,20 +24,20 @@ jobs:
     runs-on: ubuntu-latest
     needs: tests
     steps:
-      - uses: actions/checkout@v4
-      - name: Set up Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: '3.x'
-      - name: Install build tool
-        run: pip install build
-      - name: Build a binary wheel and a source tarball
-        run: python -m build --sdist --wheel
-      - name: Store the distribution packages
-        uses: actions/upload-artifact@v4
-        with:
-          name: python-package-distributions
-          path: dist/
+    - uses: actions/checkout@v4
```

Comment:
Why this indentation change?
I realize we use a mix of styles, but I've trying to move to the more standard indentation where the list items are themselves indented wrt the list heading, which you've just undone here.

Reply:
I dunno, some automated tool did it for me.  Perhaps a version of pre-commit something-or-other changed?

I don't at all care one way or the other...